modules/channel-web/src/views/lite/core/api.tsx

```diff
@@ -135,7 +135,11 @@ export default class WebchatApi {
 
   async sendMessage(payload: any, conversationId: number): Promise<void> {
     try {
-      return this.axios.post('/messages', { userId: this.userId, conversationId, payload }, this.axiosConfig)
+      return this.axios.post(
+        `/messages/${encodeURIComponent(window.__BP_VISITOR_SOCKET_ID)}`,
```

Comment:
Aren't socket ids already URI-Safe ? I didn't check the generation method

Reply:
I wouldn't have write this if they were.